### ISSUE TITLE
Breaking VariantRecord changes under 0.11.2.1

### ISSUE THREAD

**Issue by msto** [pysam-developers/pysam#479]
Hi,

Since updating to 0.11.2.1, I can no longer use the `VariantFile.new_record()` constructor for `VariantRecord`s. I receive the following error:

```
    std_rec = self.std_vcf.new_record()
  File "pysam/libcbcf.pyx", line 4282, in pysam.libcbcf.VariantFile.new_record (pysam/libcbcf.c:64486)
  File "pysam/libcbcf.pyx", line 2040, in pysam.libcbcf.VariantHeader.new_record (pysam/libcbcf.c:30838)
ValueError: At least one non-empty reference allele must be specified
```

The code works correctly under 0.11.

Additionally, it appears that reading from the END INFO field is no longer permitted.

```
    posB = record.info['END']
  File "pysam/libcbcf.pyx", line 2521, in pysam.libcbcf.VariantRecordInfo.__getitem__ (pysam/libcbcf.c:38363)
KeyError: 'END is a reserved attribute; access is via record.stop'
```

Finally, when I try to set a record's END under the new API, I cannot specify an END less than the start position. Could this requirement please be lifted? It prohibits interchromosomal breakend records where the END position is independent of the record's POS. 

```
> record.stop = 123
*** ValueError: Stop coordinate must be greater than or equal to start
```

Please let me know if I can supply additional information that would be helpful. Thanks for looking into this!

Matt

**Comment by bioinformed** [comment]
Thanks for the reports.  There were indeed large changes to the `VariantRecord` API. 

Re `At least one non-empty reference allele must be specified`:

* This limitation can be removed.

Re `KeyError: 'END is a reserved attribute; access is via record.stop'`

* This is an intentional feature.

Re `KeyError: 'END is a reserved attribute; access is via record.stop'`

* I'll look into lifting this limitation.   In htslib, this will require verifying that it is safe to store a negative reference length.

**Comment by bioinformed** [comment]
Resolved in release v0.11.2.2

**Comment by bioinformed** [comment]
@msto: Please let me know if this new release solves your problems.  Thanks!
